### ISSUE TITLE
NPCs get stuck in the corners of buildings

### ISSUE THREAD

**Issue by Hubcapp** [Open-RSC/Game#1792]
**Describe the bug**
I believe that aggressive NPCs get lured to a location not normally in their pathing distance, then refuse to walk even further away from their normal "home" in order to get back. So they get stuck in the corners of buildings b/c they don't want to go even further away from "home" to walk out the door. Never saw this phenomenon in JagexRSC to the extent that it's a "problem" in OpenRSC (check ardougne square for example of what I mean)

More info needed on how NPCs would un-stuck themselves. An element of this could be authentic; workman gets stuck in corner of draynor bank authentically, but he was really far from home and couldn't possibly path back. See a lot more clumping in OpenRSC than feels normal.

**Comment by twothousandandone** [comment]
I agree that there seems to be something about walking behavior that is not currently handled with ORSC.

This does seem to lead to more "sticking" of NPC's in areas

**Comment by Hubcapp** [comment]
In RS2, "stuck" NPCs just walk through walls at some point, in order to get back home. That's just trivia though, not sure how RSC handles it.

**Comment by kInGkRiStOf** [comment]
Zephyr would be the most reliable source for finding these mechanics. From my experience, the only way to unstick them, is to kill them. Additionally, ORSC gets weekly or more server refreshes.
